diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -107,6 +116,24 @@ open class DiktatJavaExecTaskBase @Inject constructor(
     @Suppress("FUNCTION_BOOLEAN_PREFIX")
     override fun getIgnoreFailures(): Boolean = ignoreFailuresProp.getOrElse(false)
 
+    private fun createReporterFlag(diktatExtension: DiktatExtension): String {
+        val flag: StringBuilder = StringBuilder()
+
+        // Plain, checkstyle and json reporter are provided out of the box in ktlint
+        when(diktatExtension.reporterType) {
+            "json" -> flag.append("--reporter=json")
+            "html" -> flag.append("--reporter=html")
+            "checkstyle" -> flag.append("--reporter=checkstyle")
+            else -> flag.append("--reporter=plain")
+        }
+
+        if (diktatExtension.output.isNotEmpty()) {
+            flag.append(",output=${diktatExtension.output}")
```

Comment:
Does `plain` support this option too? If not, it might need validation here.

Reply:
Yes, it supports it as well